testbenches/cosimulation.mk

```diff
@@ -55,7 +55,7 @@ $(VANILLA_LOG_RULES): SIM_ARGS +=+trace
 $(VANILLA_LOG_RULES): %.vanilla.log: $(EXEC_PATH)/%.log
```

Comment:
`VANILLA_LOG_RULES` -> `VANILLA_CSV_RULES`
%.vanilla.log -> `%.vanilla.csv`

(you should test these to make sure they work appropriately)